test/snapshots/test/index.test.ts.md

```diff
@@ -63,14 +63,13 @@ Generated by [AVA](https://avajs.dev).
 
 ## should insert CSS into head tag
 
-> Snapshot 1
+> actual_a content is compiled with insertStyle
 
-    `import ___$insertStyle from '../../../src/insertStyle.js';␊
-    ␊
-    ___$insertStyle("body{color:red}");␊
-    ␊
-    ___$insertStyle("body{color:green}");␊
-    `
+    'insertStyle("body{color:red}");'
+
+> actual_b content is compiled with insertStyle
+
+    'insertStyle("body{color:green}");'
```

Comment:
Is `...insertStyle` made global?  Is this why the import is not showing here?

Reply:
This snaposhot is testing the value of `code` property of a rollup `RenderedModule` so it won't include any import statement:

https://github.com/marcalexiei/rollup-plugin-sass/blob/d372283ed53837549e73214497ee1d4d6f9f8871/test/index.test.ts#L169-L177